ui/lib/pki/addon/components/pki-config/import.hbs

```diff
@@ -1,14 +1,16 @@
-<form {{on "submit" this.submitForm}} data-test-pki-config-import-form>
+<form {{on "submit" this.handleSave}} data-test-pki-config-import-form>
```

Comment:
I think we can use the same component used in to import an issuer? https://github.com/hashicorp/vault/pull/18634/files#diff-d6a7836ae23bb59f176863c19f413483b08c82a3f2dc81770b445d4a021fdd39

I originally named it 'import-ca-certificate' thinking it could be reused here, but now that this form no longer handles unloading the record it'll have to be refactored a bit. What are your thoughts?